requirements/dev.txt

```diff
@@ -17,7 +17,7 @@ Scrapy==1.4.0
 coverage==4.4.1
 
 # fake for LDAP testing
--e git+https://github.com/zulip/fakeldap.git@37bc199e9589c31f4bb5bae1445ad1cd7f84e1ac#egg=fakeldap==0.5.1.zulip1
+-e git+https://github.com/zulip/fakeldap.git@0.6.1#egg=fakeldap==0.6.1.zulip1
```

Comment:
We don't need the `#egg` part or Git URL.  This should just become `fakeldap==0.6.1`, since it's been properly released :)